src/app/(home)/[not-found]/page.module.css

```diff
@@ -0,0 +1,111 @@
+.notFoundPageWrap {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 100vw;
+	height: 100vh;
+	background-color: var(--color-silver-gray);
+}
+.notFoundPageContainer {
+	display: flex;
+}
+.notFoundPageContainer__image {
+	width: 409px;
+	height: 407px;
+}
+.titleContainer {
+	font-family: "Rubik";
```

Comment:
this line is unnecessary because Rubik is default font (mostly used) and was connected in src/app/globals.css